Features/Context/PlatformUI.php

```diff
@@ -0,0 +1,198 @@
+<?php
+
+/**
+ * File containing the main context class for PlatformUI
```

Comment:
I'd stick to the standard header ("This file is part of the EzSystemsBehatBundle package."), and document the class in the class's phpdoc (it is useful, and it will make it more visible in IDEs).